soda/core/soda/execution/data_source.py

```diff
@@ -242,6 +247,154 @@ def sql_information_schema_identifier(self) -> str:
     def sql_analyze_table(self, table: str) -> str | None:
         return None
 
+    def profiling_sql_values_frequencies_query(self, table_name: str, column_name: str) -> str:
+        return dedent(
+            f"""
+                with freq_values AS (
```

Comment:
Can you change the SQL keywords to uppercase (like in other places in the code)?

Reply:
Will do. I imagine that's a convention we want to be consistent about? Because I am not a fan of uppercase in SQL however if it is a convention I'll comply.

Reply:
Since in other places we use capitals already - I think it would be good to keep the same - because these queries end up in the log and can be a bit annoying for the users if the formatting is inconsistent.